spec/paper_trail/model_spec.rb

```diff
@@ -264,11 +264,18 @@
           expect(widget.versions.last.item).to be_nil
         end
 
-        it "not have changes" do
-          widget = Widget.create(name: "Henry")
-          widget.update_attributes(name: "Harry")
-          widget.destroy
-          expect(widget.versions.last.changeset).to eq({})
+        it "has changes" do
+          book = Book.create! title: "A"
+          changes = YAML.load book.versions.last.attributes["object_changes"]
+          expect(changes).to eq "id" => [nil, book.id], "title" => [nil, "A"]
+
+          book.update! title: "B"
+          changes = YAML.load book.versions.last.attributes["object_changes"]
+          expect(changes).to eq "title" => %w[A B]
+
+          book.destroy
+          changes = YAML.load book.versions.last.attributes["object_changes"]
+          expect(changes).to eq "id" => [book.id, nil], "title" => ["B", nil]
```

Comment:
This test looks sufficient, but it'd be great to also have unit tests in eg. `spec/paper_trail/events/destroy_spec.rb`. Not a requirement, use your judgement.

Reply:
It doesn't look like the unit tests for the various events currently cover `object` or `object_changes`. There's an argument to be made that a lot of the tests in `mode_spec.rb` should be moved out, but I think that's beyond the scope of this PR. This test is probably enough for now.